WordPress/src/main/java/org/wordpress/android/ui/compose/components/shimmer/Shimmer.kt

```diff
@@ -0,0 +1,56 @@
+package org.wordpress.android.ui.compose.components.shimmer
+
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.unit.IntSize
+import org.wordpress.android.ui.compose.theme.AppColor
+
+object Shimmer {
+    val color = AppColor.Black.copy(
+        alpha = 0.08F
+    )
+}
+
+@Suppress("MagicNumber")
+fun Modifier.shimmerLoadingAnimation(): Modifier = composed {
+    var size by remember {
+        mutableStateOf(IntSize.Zero)
+    }
+    val transition = rememberInfiniteTransition()
+    val startOffsetX by transition.animateFloat(
+        initialValue = -2 * size.width.toFloat(),
+        targetValue = 2 * size.width.toFloat(),
+        animationSpec = infiniteRepeatable(
+            animation = tween(1000)
+        ),
+        label = "Shimmer animation",
+    )
+
+    background(
+        brush = Brush.linearGradient(
+            colors = listOf(
+                Color(0xFFB8B5B5),
+                Color(0xFF8F8B8B),
+                Color(0xFFB8B5B5),
+            ),
+            start = Offset(startOffsetX, 0f),
+            end = Offset(startOffsetX + size.width.toFloat(), size.height.toFloat())
+        )
+    )
+        .onGloballyPositioned {
+            size = it.size
```

Comment:
Should we check if the size has changed  to prevent unnecessary recompositions? something like 
```
.onGloballyPositioned { layoutCoordinates ->
    val newSize = layoutCoordinates.size
    if (size != newSize) {
        size = newSize
    }
}
```